package.json

```diff
@@ -95,6 +93,7 @@
     "warp-wasm-metering": "1.0.1"
   },
   "devDependencies": {
+    "@idena/vrf-js": "^1.0.1",
```

Comment:
? what for

Reply:
for tests :-) https://github.com/warp-contracts/warp/pull/401/files#diff-fc8f73adefb2d436efca49ad1f9867b9ba3e40610db08a6c574ccbb92565ca74R22